eng/azure-sdk-deps.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <EnableDefaultItems>false</EnableDefaultItems>
+    <PublishDepsFilePath>$(IntermediateOutputPath)\azure-sdk-deps.json</PublishDepsFilePath>
```

Comment:
"azure-sdk.deps.json"? You have a "dash" instead of a "dot", which is what the old file was and is what you have in your PR title.

Reply:
It still has the same output file name in the pipelines file but your correct I can also update this to match for consistency.